src/fetchMetals.ts

```diff
@@ -16,9 +16,10 @@ export function fetchMetals({
     (p) => !p.startsWith("-agentlib")
   );
 
+  const binaryVersion = serverVersion > "0.11.2" ? "2.13" : "2.12";
```

Comment:
This comparison seems a bit terrifying, but I guess alphabetical order is sufficient here.

Reply:
Oops, it's not ok for `0.11.2-RC1` or something

Reply:
I see, metals-languageclient has `semver` as dependency and we can use that :+1: